pythia/tasks/captioning/coco/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
The header message seems incomplete here

Reply:
Different files have different headers. Which is the correct one we should use?

Reply:
This one works. It is short form of the original header used everywhere.